cmd/cluster/aws/create.go

```diff
@@ -156,6 +156,18 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 		tags = append(tags, hyperv1.AWSResourceTag{Key: k, Value: v})
 	}
 
+	var instanceType string
+	if opts.AWSPlatform.InstanceType != "" {
+		instanceType = opts.AWSPlatform.InstanceType
+	} else {
+		switch opts.Arch {
+		case "amd64":
+			instanceType = "m5.large"
+		case "arm64":
+			instanceType = "m6g.large"
```

Comment:
can you add a comment with the source/reasoning for this default?

Reply:
Done